configuration.md

```diff
@@ -63,64 +72,85 @@ Here is a specimen of a JSON configuration file:
 ```json
 {
   "port": 80,
-  "noAuth": false,
   "baseURL": "/admin",
   "address": "127.0.0.1",
-  "alternativeReCaptcha": false,
-  "reCaptchaKey": "",
-  "reCaptchaSecret": "",
   "database": "/path/to/database.db",
   "log": "stdout",
   "plugin": "",
-  "scope": "/path/to/my/files",
-  "allowCommands": true,
-  "allowEdit": true,
-  "allowNew": true,
-  "commands": [
-    "git",
-    "svn"
-  ]
+  "auth": {
+    "method": "default",
+    "header": ""
```

Comment:
Empty fields are required or did you add them to let users know that the option exists?